styles.css

```diff
@@ -0,0 +1,795 @@
+    color: #000;
+    padding-top: 15px;
+  }
+
+  .nav-link:hover {
+    color: #1422bb;
+  }
+
+  .nav-menu.active {
+    left: 0;
+  }
+
+  #logo {
+    display: none;
+  }
+
+  header {
+    display: none;
+  }
+
+  .programm {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #212121;
+  }
+
+  .line-programm {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #212121;
+  }
+
+  .main-programm {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 90%;
+    background-color: #424242;
+    color: white;
+    margin: 10px;
+    padding-left: 5px;
+    padding-right: 5px;
+    font-size: 16px;
+  }
+
+  .under-line {
+    height: 5px;
+    width: 60px;
+    background-color: red;
+  }
+
+  .main-section {
+    margin: 15px;
+    height: auto;
+    background-image: url('assets/white-technology-background-circuit-diagrams_115579-1036.webp');
+    background-repeat: no-repeat;
+    background-size: 300px 400px;
+  }
+
+  h1 {
+    font-size: 18px;
+    color: #ff6347;
+    margin-bottom: 0;
+    margin-top: 40px;
+  }
+
+  h5 {
+    font-size: 35px;
+    margin-top: 0;
+    background: -webkit-linear-gradient(#f00, #333);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+  }
+
+  #first-para {
+    color: #000;
+    font-size: 16px;
+  }
+
+  #sec-para {
+    font-size: 25px;
+    margin-bottom: 0;
+  }
+
+  #third-para {
+    font-size: 20px;
+    margin-top: 0;
+  }
+
+  #card-theme {
+    margin: 10px;
+  }
+
+  .patterns {
+    width: 20vw;
+    height: 20vw;
+    margin-left: 10px;
+  }
+
+  .pt1 {
+    background-color: #eee;
+    background-size: 30px 30px;
+    background-position: 0 0, 30px 30px;
+    background-image: -webkit-linear-gradient(45deg, black 25%, transparent 25%, transparent 75%, black 75%, black), -webkit-linear-gradient(45deg, black 25%, transparent 25%, transparent 75%, black 75%, black);
+    background-image: -moz-linear-gradient(45deg, black 25%, transparent 25%, transparent 75%, black 75%, black), -moz-linear-gradient(45deg, black 25%, transparent 25%, transparent 75%, black 75%, black);
+    background-image: -ms-linear-gradient(45deg, black 25%, transparent 25%, transparent 75%, black 75%, black), -ms-linear-gradient(45deg, black 25%, transparent 25%, transparent 75%, black 75%, black);
+    background-image: -o-linear-gradient(45deg, black 25%, transparent 25%, transparent 75%, black 75%, black), -o-linear-gradient(45deg, black 25%, transparent 25%, transparent 75%, black 75%, black);
+    background-image: linear-gradient(45deg, black 25%, transparent 25%, transparent 75%, black 75%, black), linear-gradient(45deg, black 25%, transparent 25%, transparent 75%, black 75%, black);
+  }
+
+  #img-feature {
+    width: 90px;
+    height: 100px;
+    margin-left: 10px;
+    border: 10px solid #b2beb5;
+  }
```

Comment:
Per the design there should be no border around the image of the speakers. Kindly remove that.

Your design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-20 09-15-26](https://user-images.githubusercontent.com/65233860/196908348-6b688dbe-a4d6-4936-a71f-cea88214f924.png)  |  ![Screenshot from 2022-10-20 09-15-43](https://user-images.githubusercontent.com/65233860/196908389-43e6ffaf-05da-498b-b724-b4e6acddab64.png)